dd-sdk-android/src/main/kotlin/com/datadog/android/telemetry/internal/Telemetry.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.telemetry.internal
+
+import com.datadog.android.rum.GlobalRum
+import com.datadog.android.rum.internal.monitor.AdvancedRumMonitor
+import com.datadog.android.rum.internal.monitor.NoOpAdvancedRumMonitor
+
+internal class Telemetry {
```

Comment:
Why do you need this extra class and not call `RumMonitor` directly in the `TelemetryLogHandler` ?

Reply:
because per RFC it is supposed also to use `telemetry` for reporting configuration and, maybe, performance metrics, so generally telemetry != logs.